src/main/java/me/totalfreedom/totalfreedommod/command/Command_tag.java

```diff
@@ -12,18 +14,17 @@
 import org.bukkit.entity.Player;
 
 @CommandPermissions(level = Rank.OP, source = SourceType.BOTH)
-@CommandParameters(description = "Sets yourself a prefix", usage = "/<command> <set <tag..> | off | clear <player> | clearall>")
+@CommandParameters(description = "Sets yourself a prefix", usage = "/<command> <set <tag> | off | clear <player> | clearall | list>")
 public class Command_tag extends FreedomCommand
 {
 
-    public static final List<String> FORBIDDEN_WORDS = Arrays.asList(new String[]
-    {
-        "admin", "owner", "moderator", "developer", "console"
-    });
+    public static final List<String> FORBIDDEN_WORDS = Lists.newArrayList();
 
     @Override
     public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
     {
+        FORBIDDEN_WORDS.addAll((Collection<? extends String>) ConfigEntry.BLOCKED_TAGS.getList());
```

Comment:
Ouch. This will not go over well. Memory leak right here.

Reply:
Changes have now been made to fix this. If you're happy with the changes I'd love to get this merged in and a release done, then get the new version underway.